src/observable.ts

```diff
@@ -16,8 +16,8 @@ interface Observable<T> {
   subscribe(observer: ObservableObserver<T>): ObservableSubscription;
 }
 
-const observableSymbol = (): symbol =>
-  (Symbol as any).observable || ((Symbol as any).observable = Symbol('observable'));
+const observableSymbol = (): symbol | string =>
+  (typeof Symbol === 'function' && Symbol.observable) || '@@observable';
```

Comment:
```suggestion
  Symbol.observable || '@@observable';
```

`wonka`, as per its most common usage in `urql`, actually makes some minimum assumptions now as per the browser support that's required. With the departure of IE11 in what's supported, it's currently assumed that support looks like:

```
last 3 ios_saf versions
last 3 and_chr major versions
op_mini > 99
and_uc > 99
last 3 samsung major versions
last 3 opera versions
last 3 safari versions
last 10 chrome major versions
last 8 firefox major versions
ie > 99
last 3 edge versions
```

hence, `Symbol` support can be assumed, and it being missing in the global scope should fail.